docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -131,17 +132,22 @@ Use the {product-long-kafka} web console to create and configure a Kafka instanc
 ifndef::qs[]
 .Prerequisites
 * You're logged in to the {product-kafka} web console at {service-url-kafka}[^].
+* You have a registered {rosa-name} cluster in a ready state.
 * You're familiar with the different Kafka instance sizes that you can create in {product-kafka}, and the capacities that correspond to those sizes. For more information about the capacities and service limits of your Kafka instance, see https://access.redhat.com/articles/5979061[Red Hat OpenShift Streams for Apache Kafka Service Limits].
 endif::[]
 
 .Procedure
 . On the *Kafka Instances* page of the {product-kafka} {service-url-kafka}[web console^], click *Create Kafka instance*.
 . Complete the *Create a Kafka instance* form to define the following instance details. Some instance details currently have only one option.
 * *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-* *Cloud provider*: For a production Kafka instance, accept the default value of *Amazon Web Services*. If you're creating a trial instance, you also have the option to select *Google Cloud Platform*.
-* *Cloud region*: Select a cloud provider region from the list.
```

Comment:
These options no longer apply. Also from section 6.4 of the MAS Style Discussion guide, we agreed to use 'list' instead of 'drop-down' to conform with IBM standards.